modules/gapi/misc/python/package/gapi/__init__.py

```diff
@@ -249,3 +260,13 @@ def kernel_with_params(cls):
         return cls
 
     return kernel_with_params
+
+
+# FIXME: On the c++ side every class is placed in cv2 module.
+cv.gapi.wip.draw.Rect = cv.gapi_wip_draw_Rect
```

Comment:
@alalek How ugly is it ? Is there another option ?

Reply:
Not sure why you need that. There are many other classes from nested namespaces which are located properly.

AFAIK, there is some bug with Python ctors (this is why "create" method is used).
It is better to fix that problem instead of replacing classes with functions.

Reply:
>>> Not sure why you need that. There are many other classes from nested namespaces which are located properly.
Could you provide any examples ? Looks like all `classes` is placed in `cv2` module.

Reply:
From generated file:
```
CVPY_TYPE(gapi_wip_draw_Rect, gapi_wip_draw_Rect, cv::gapi::wip::draw::Rect, Rect, NoBase, pyopencv_cv_gapi_wip_draw_gapi_wip_draw_Rect_Rect)
```